ktor-server/ktor-server-jetty/ktor-server-jetty-test-http2/build.gradle

```diff
@@ -18,9 +18,9 @@ configure(tasks.jvmTest) {
     exclude("**/*StressTest*")
 
     if (JavaVersion.current() == JavaVersion.VERSION_1_8) {
-        jvmArgs {
-            def bootClasspath = configurations.boot.files
-            bootClasspath.collect { "-Xbootclasspath/p:${it.absolutePath}" }.iterator()
-        }
+       // jvmArgs {
```

Comment:
?

Reply:
that should not have gone into the commit, there was an error preventing me from building.

Reply:
that should not have gone into the commit, there was an error preventing me from building. fixed with the next push